Sources/Controllers/DashboardOnMap/MapSettings/MapSettingsGpxViewController.swift

```diff
@@ -459,8 +495,246 @@ final class MapSettingsGpxViewController: OABaseNavbarSubviewViewController {
         }
     }
     
+    private func setupHeaderView() -> UIView? {
+        let headerView = UIView(frame: CGRect(x: 0, y: 0, width: tableView.frame.width, height: 44))
```

Comment:
The use of .init() is highlighted by SwiftLint as an 'Explicit Init Violation' because explicitly calling .init() when initializing objects in Swift is generally discouraged for better code readability. Swift's syntax allows us to omit .init() for a cleaner and more concise code style.

Reply:
UIView(frame: .init(x: 0, y: 0, width: tableView.frame.width, height: 44)) - warning?

Reply:
Ok!